src/main/java/com/aws/greengrass/util/platforms/windows/WindowsPlatform.java

```diff
@@ -120,13 +126,33 @@ protected void setPermissions(FileSystemPermission permission, Path path,
     }
 
     @Override
-    public UserAttributes lookupUserByName(String user) throws IOException {
-        return null;
+    public WindowsUserAttributes lookupUserByName(String user) throws IOException {
```

Comment:
This is only used to determine if a user exists during setup; not sure we actually need this to be in the platform abstraction. If we can get rid of these, then I'd like that in order to remove the extra dependency.

Reply:
I am a little hesitate to remove this method since I haven't look into the setup code yet. I worried about if we remove this method now, we may need it when we work on the setup code for Windows. What do you think?

Reply:
The method we need is userExists and not really this method.